ark/analysis/spatial_analysis.py

```diff
@@ -58,21 +59,20 @@ def calculate_channel_spatial_enrichment(dist_matrices_dict, marker_thresholds,
                              "eccentricity", "major_axis_length", "minor_axis_length",
                              "perimeter", "fov"]
 
-    # Error Checking
-    if not np.isin(excluded_colnames, all_data.columns).all():
-        raise ValueError("Column names were not found in Expression Matrix")
+    # check if included fovs found in fov_col
+    misc_utils.verify_in_list(fov_names=included_fovs,
+                              unique_fovs=all_data[fov_col].unique())
 
-    if not np.isin(included_fovs, all_data[fov_col]).all():
-        raise ValueError("Fovs were not found in Expression Matrix")
+    # check if all excluded column names found in all_data
+    misc_utils.verify_in_list(columns_to_exclude=excluded_colnames,
+                              column_names=all_data.columns)
 
     # Subsets the expression matrix to only have channel columns
     all_channel_data = all_data.drop(excluded_colnames, axis=1)
 
-    # this will get refactored once the verification refactor PR gets merged in
-    if not np.all(set(marker_thresholds.iloc[:, 0]) == set(all_channel_data.columns)):
-        raise ValueError(
-            "The same markers must be found in marker thresholds and expression matrix columns"
-        )
+    # check that the markers are the same in marker_thresholdsa and all_channel_data
+    misc_utils.verify_same_elements(markers_to_threshold=marker_thresholds.iloc[:, 0].values,
```

Comment:
`markers_to_threshold` is a perfect name. `all_data_marker_columns` not as great. Maybe `all_markers`, 'marker_names`, `column_names`, etc